tests/common_test.py

```diff
@@ -373,6 +374,105 @@ def fun(w, X, y):
         msg = "weak_type inconsistency for attribute '%s' in solver '%s'"
         self.assertEqual(weak_type0, weak_type1, msg=msg % (field, solver_name))
 
+  def test_jit_with_verbose(self):
+
+    fun = lambda p: p @ p
+    fun_vec_out = lambda p: p**2 - p
+
+    def fixed_point_fun(params):
+      return fun(params) - params
+
+    solvers = (
+      # Unconstrained
+      jaxopt.GradientDescent(fun=fun, jit=True, verbose=1),
+      jaxopt.PolyakSGD(fun=fun, jit=True, verbose=1),
+      jaxopt.OptaxSolver(opt=optax.adam(1e-3), jit=True, fun=fun),
+      jaxopt.Broyden(fun=fun_vec_out, jit=True),
+      jaxopt.AndersonAcceleration(fixed_point_fun=fixed_point_fun, jit=True, verbose=True),
+      jaxopt.ArmijoSGD(fun=fun, jit=True, verbose=1),
+      jaxopt.BFGS(fun, linesearch="zoom", jit=True, verbose=1),
+      jaxopt.BFGS(fun, linesearch="backtracking", jit=True, verbose=1),
+      jaxopt.BFGS(fun, linesearch="hager-zhang", jit=True, verbose=1),
+      jaxopt.LBFGS(fun=fun, jit=True, verbose=1),
+      jaxopt.ArmijoSGD(fun=fun, jit=True, verbose=1),
+      jaxopt.NonlinearCG(fun, jit=True, verbose=1),
+      # Unconstrained, nonlinear least-squares
+      jaxopt.GaussNewton(residual_fun=fun, jit=True, verbose=True),
+      jaxopt.LevenbergMarquardt(residual_fun=fun, jit=True, verbose=True),
+      # Constrained
+      jaxopt.ProjectedGradient(fun=fun,
+        projection=jaxopt.projection.projection_non_negative, jit=True, verbose=1),
+      # Optax wrapper
+      jaxopt.OptaxSolver(opt=optax.adam(1e-1), fun=fun, jit=True, verbose=1),
+    )
+
+    @partial(jax.jit, static_argnums=(1,))
+    def f(p0, solver):
+        return solver.run(p0)
+
+    for solver in solvers:
+      f(jnp.arange(2.), solver)
+
+    # Proximal gradient solvers
+    fun = objective.least_squares
+    X, y = datasets.make_classification(n_samples=10, n_features=5, n_classes=3,
+                                        n_informative=3, random_state=0)
+    data = (X, y)
+    params0 = jnp.zeros(X.shape[1])
+
+    @partial(jax.jit, static_argnums=(1,))
+    def f_prox(p0, solver):
+      update = jax.jit(solver.update)
+      state0 = solver.init_state(p0, hyperparams_prox=1.0, data=data)
+      _, state = update(p0, state0, hyperparams_prox=1.0, data=data)
+      return state0, state
+    
+    for solver in (jaxopt.ProximalGradient(fun=fun, prox=prox.prox_lasso),
+                   jaxopt.BlockCoordinateDescent(fun=fun, block_prox=prox.prox_lasso)):
+      state0, state = f_prox(params0, solver)
+      check_states_have_same_types(state0, state)
```

Comment:
Remove this line (the goal of this test is to test jittability with verbose=True, not to check types)